sphere-java-client-ahc-1_9/src/it/java/io/sphere/sdk/client/CustomClientConfigDemoTest.java

```diff
@@ -0,0 +1,46 @@
+package io.sphere.sdk.client;
+
+import com.ning.http.client.AsyncHttpClient;
+import com.ning.http.client.AsyncHttpClientConfig;
+import io.sphere.sdk.http.*;
+import org.junit.Test;
+
+import java.util.concurrent.CompletionStage;
+
+import static io.sphere.sdk.http.HttpMethod.GET;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CustomClientConfigDemoTest {
+    /**
+     * Creates a {@link HttpClient} that can be used by the {@link SphereClient}.
+     * @return new http client with custom settings
+     */
+    public static HttpClient createCustomHttpClient() {
+        final AsyncHttpClientConfig httpClientConfig = new AsyncHttpClientConfig.Builder()
+                .setEnabledProtocols(new String[]{"TLSv1.2"})//required
+                //some examples what can be configured
+                .setMaxConnections(500)
+                .setConnectTimeout(10)
+//                .setProxyServer(proxy)
+                .build();
+        final AsyncHttpClient asyncHttpClient = new AsyncHttpClient(httpClientConfig);
+        return AsyncHttpClientAdapter.of(asyncHttpClient);
+    }
+
+    /**
+     * Shows the initialization of a {@link SphereClient} with a custom {@link HttpClient} provider.
+     */
+    private void demoCreateClient() {
```

Comment:
Is this method related to the other method or the test? Because I guess it's just to show how to use it in the sphere client, but that confuses a bit, I think. I cannot think of any suggestion to improve it though, sorry.